### PR TITLE
Diagram based database

### PR DESCRIPTION
**Pull request by Samuellekpor**
In this milestone, we did the following:

1. Create the clinic database
2. Add tables based on the provided diagram

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+CREATE TABLE invoice_items(
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    unit_price DECIMAL,
+    quantity INT,
+    total_price INT,
+    invoice_id INT,
+    FOREIGN KEY invoice_id REFERENCES invoices(id),
+    treatment_id INT,
+    FOREIGN KEY treatment_id REFERENCES treatments(id)
+)
```
Comment (thecodechaser) [inline schema_based_on_diagram.sql:60]: - [ ] Please make sure to add the FOREIGN KEYS indexes as requested in the project requirements.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Samuellekpor, and @kaizipaul👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (rloterh) [reply schema_based_on_diagram.sql:60]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Team,  

Your project is complete! There is nothing else to say other than... it's time to merge it. Congratulations!  🎉🌟 

## Highlights 👍
- Database properly established!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
- All assigned (3) members to this project are required to contribute evenly to this project. Kindly ensure to enforce this in your next assigned group project.
- Kindly describe or include only the tech stacks and sections that are relevant to your project. 
- Provided text instructions should be removed and filled in with custom content applicable to your project.
- Kindly ensure to remove redundant demo links in your ReadMe.
- Kindly describe between 1-3 key features of the application.
- Kindly ensure to add the _Future Features_ section to your ReadMe, which should include 1-3 potential features to be added.
- MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove the guide note below it.

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**